compiler/executable/match_/planner/plan.rs

```diff
@@ -575,8 +575,8 @@ impl<'a> ConjunctionPlanBuilder<'a> {
                 Comparator::NotEqual => (), // no tangible impact on traversal costs
                 Comparator::Less | Comparator::LessOrEqual => rhs.add_upper_bound(lhs),
                 Comparator::Greater | Comparator::GreaterOrEqual => rhs.add_lower_bound(lhs),
-                Comparator::Like => unimplemented_feature!(ComparatorLike),
-                Comparator::Contains => unimplemented_feature!(ComparatorContains),
+                Comparator::Like => (),
+                Comparator::Contains => (),
```

Comment:
`like` and `contains` don't reduce the total cost, but they impact the `io_ratio` by filtering out rows. The planner should take that into account.
This is probably out of scope here, but warrants at least a TODO comment.